pallets/schemas/README.md

```diff
@@ -0,0 +1,79 @@
+# Schemas Pallet
+
+The Schemas Pallet provides universal schema registration for data flowing through Frequency.
+
+## Summary
+
+This pallet provides an on chain repository for schemas, thereby allowing participants of the network to flexibly interact and exchange messages with each other with the correct human intent and data structure.
+All Messages and Stateful Storage content must be attached to a Schema Identifier so that the content can be correctly parsed and interpreted.
+
+### Data Structure vs Human Intent
+
+Schemas provide for both consistent data structures, but also human intent of the message.
+Some schemas may be structurally the same, but have a different interpretation of the contents.
+For example, two schemas might both only have a hash for contents, but one is a recording of the hash for time validation purposes, while the other is to mark an off-chain vote.
+
+### Schema Parameters
+
+- Model: The actual JSON representing the data structure.
+- Model Type: The type of serialization used. (Parquet, Avro, etc...)
+- Settings: Various options for the Schema like signature requirements.
```

Comment:
Perhaps elaborate on available settings? There are only a couple...

Reply:
@JoeCap08055 / @saraswatpuneet added a whole section each on Type, Settings, and Payload Location.

Reply:
<img width="1113" alt="image" src="https://github.com/frequency-chain/frequency/assets/1252199/d94e34ff-d73b-4c57-b022-cbd30f3fc7ee">